lib/logger/logger.go

```diff
@@ -62,6 +62,12 @@ func loadLoggerConfig() (*viper.Viper, error) {
 	cfg.SetConfigType("yaml")
 	cfg.SetConfigName("logger")
 	cfg.AddConfigPath("data/config")
+	cfg.SetEnvPrefix("GSW_LOGGER")
+	cfg.AutomaticEnv()
```

Comment:
same conmment as above. maybe im missing something, but why does this need to become AutomaticEnv when it seems like all our uses of the configs happen once at the start

Reply:
Here AutomaticEnv is less useful because I use BindEnv (AutomaticEnv struggles with the camel case) but it's still nice to have (re: above).